snmp-notifier/templates/app-configmap.yaml

```
@@ -0,0 +1,41 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: {{include "notifier.fullName" .}}-app-config
+    namespace: {{.Release.Namespace}}
+    labels:
+{{ include "notifier.labels" . | indent 8 }}
+    annotations:
+        com.dellemc.kahm.subscribed: "true"
+data:
+    eventRules: |-
+      issueRules:
+      - description: "SNMP connection issue"
+        name: SNMPConnection
+        issueCategory: Auto
+        matchOnList:
+         - matchon:
+           - label: SymptomID
+             value: SNMPNOT-1 
+        notifiers:
+
+      rules:
+      - description: test SNMP TRAP
+        matchon:
+        - field: type
+          value: Normal
+        - label: SymptomID 
+          value: TEST-TRAP
+        notifiers:
+        - {{.Values.product}}-snmp-notifier
+    eventRemedies: |
+      symptoms:
+        - symptomid: SNMPNOT-1
+          description: Dell EMC SNMP Notifier is not able to send TRAPs due to connection or authetication failure for product {{.Values.product}}
+          remedies:
+            - Check if the SNMP user is configured with the correct authetication parameters
+            - Check if the engineID is matching with the engineID configured for the user in SNMP manager
+            - Check if the SNMP notifier is configured with the correct SNMP manager host/IP address and port
+            - Check if the community string is configured correct in case of v2c SNMP manager
```

is it server or Manager?  
```suggestion
            - Verify the SNMP credentials are configured with the correct authetication values
            - Verify thee engineID matches with the engineID configured for the product in the SNMP server
            - Verify the product SNMP notifier is configured with the correct host/IP address and port
            - Verify connectivity to the SNMP server, check firewalls, network routing etc...
            - Verify the SNMP v2c configuration has the correct community string
```